automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoTestUtils.scala

```diff
@@ -111,9 +115,10 @@ trait LeonardoTestUtils extends WebBrowserSpec with Matchers with Eventually wit
     // Google doesn't seem to like simultaneous cluster creates.  Add 0-30 sec jitter
     Thread sleep Random.nextInt(30000)
 
-    val cluster = Leonardo.cluster.create(googleProject, clusterName, clusterRequest)
+    val clusterTimeResult = time(Leonardo.cluster.create(googleProject, clusterName, clusterRequest))
+    val cluster = clusterTimeResult.result
     clusterCheck(cluster, googleProject, clusterName, Seq(ClusterStatus.Creating), clusterRequest)
-
+    clusterTimeResult.duration should be < (10 * nanoSecond)
```

Comment:
Could you update this to use `Duration` and `10.Seconds` (syntax may be slightly different) instead?

Reply:
Alternative: could also use `eventually` and `PatienceConfig` to do this

Reply:
Agree on using `FiniteDuration` (see above). But I'm not sure how `eventually` could be used here -- the idea is to time how long a blocking call (`Leonardo.cluster.create(...)` in this case) takes. This approach seems fine to me.

Reply:
Could you also log the latency so it's visible in test output?